packages/ui/src/ui/utils/system/masters.ts

```diff
@@ -0,0 +1,13 @@
+import {makeRegexpFromSettings} from '../../../shared/utils';
+import {uiSettings} from '../../config/ui-settings';
+
+export const calcMasterShortName = (address: string) => {
+    const re = makeRegexpFromSettings(uiSettings.reShortNameFromAddress);
```

Comment:
It is better to think that creation of RegExp is no a cheap action and create an instance once.